docs/gluetun-compose.md

```
@@ -0,0 +1,280 @@
+
+| Setting | Value |
+| --- | --- |
+| Host | `gluetun` |
+| Port | `${QBT_INT_PORT}` (or whatever WebUI port you mapped on Gluetun) |
+| Username | `${QBT_USER}` |
+| Password | `${QBT_PASS}` |
+| Category | `tv`, `movies`, `music` (optional) |
+| Enable SSL | Off unless you terminate TLS elsewhere |
+
+Example payload for Sonarr (`Settings → Download Clients → qBittorrent → Test`):
+```json
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)